src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1734,6 +1734,14 @@ def update_app_service_plan(instance, sku=None, number_of_workers=None):
     return instance
 
 
+def show_plan(cmd, resource_group_name, name):
+    client = web_client_factory(cmd.cli_ctx)
+    plan = client.app_service_plans.get(resource_group_name, name)
+    if not plan:
```

Comment:
@Kotasudhakarreddy have you tried the option we discussed in our meeting - seems like API is throwing a message with 404, but we are not caching this since CLI expects JSON response but this is giving a HTML response. I would like to avoid showing custom message & handle 404 case accurately. Thanks!

Reply:
Yes, sisira. I have tried and need to discuss with you on this.